src/api/config.ts

```diff
@@ -1,11 +1,44 @@
 import { z } from 'zod'
 import * as dotenv from 'dotenv'
+import { NetworkId } from '../types/networkId'
 
-export function getConfig() {
-  dotenv.config()
+export interface Config {
+  GET_TOKENS_INFO_URL: string
+  NETWORK_ID_TO_RPC_URL: Partial<Record<NetworkId, string>>
+  GOOGLE_CLOUD_PROJECT: string
+  POSITION_IDS: string[]
+  SHORTCUT_IDS: string[]
+}
 
+export function networkIdToRpcUrlTransform(val: string | undefined) {
+  if (!val) {
+    return {}
+  }
+  // expected format: network id/rpc url pairs joined by |, and pairs separated by spaces
+  // example: 'ethereum-mainnet|https://my-endpoint-name.quiknode.pro/my-api-key celo-mainnet|https://forno.celo.org'
```

Comment:
nit: wondering if it wouldn't be easier to have a multiline KEY=VALUE config instead? At least for editing the secret in secrets manager.

Reply:
something this?
```
ethereum-mainnet=https://my-endpoint-name.quiknode.pro/my-api-key
celo-mainnet=https://forno.celo.org
```

I didn't use equals as a separator because it's commonly used in URLs, so splitting on `'='` would break. Lmk if you were thinking of something else though

Reply:
we could certainly use newlines instead of spaces, but I used spaces because I was doing some testing in the CLI and spaces are more convenient there

Reply:
I'm going to merge for now, but lmk if you were thinking of something else and I'm happy to circle back